pkg/operator/ceph/disruption/clusterdisruption/osd.go

```diff
@@ -156,6 +157,12 @@ func (r *ReconcileClusterDisruption) reconcilePDBsForOSDs(
 
 	pgHealthMsg, clean, err := cephClient.IsClusterClean(r.context.ClusterdContext, request.Namespace)
 	if err != nil {
+		// If the error contains that message, this means the cluster is not up and running
+		// No monitors are present and thus no ceph configuration has been created
+		if strings.Contains(err.Error(), "error calling conf_read_file") {
+			logger.Infof("Ceph %q cluster not ready, cannot check Ceph status yet.", request.Namespace)
```

Comment:
can we change this to debug? no reason to fill up the logs if the cluster just isn't created yet.